browser/greaselion/greaselion_browsertest.cc

```diff
@@ -534,6 +534,27 @@ IN_PROC_BROWSER_TEST_F(GreaselionServiceTest,
   EXPECT_EQ(title, "Altered");
 }
 
+IN_PROC_BROWSER_TEST_F(GreaselionServiceTest, CleanShutdown) {
+  ASSERT_TRUE(InstallMockExtension());
+
+  GURL url = embedded_test_server()->GetURL("www.a.com", "/simple.html");
+  ui_test_utils::NavigateToURL(browser(), url);
+  content::WebContents* contents =
+      browser()->tab_strip_model()->GetActiveWebContents();
+  ASSERT_TRUE(content::WaitForLoadStop(contents));
+  EXPECT_EQ(url, contents->GetURL());
+  std::string title;
+  ASSERT_TRUE(
+      ExecuteScriptAndExtractString(contents,
```

Comment:
this function is deprecated in favor of `EvalJs`

Reply:
basically, there is `TitleWatcher` in browser_test_utils.h

Reply:
OK, I entered https://github.com/brave/brave-browser/issues/13965 for this as just about all of the tests in there will require this change.